amenity-points.mss

```diff
@@ -765,21 +769,40 @@
     marker-clip: false;
   }
 
-  [feature = 'historic_memorial'][zoom >= 17] {
+  [feature = 'historic_memorial'][memorial = null][zoom >= 17],
+  [feature = 'historic_memorial'][memorial = 'statue'][zoom >= 17],
+  [feature = 'historic_memorial'][memorial = 'bust'][zoom >= 18],
+  [feature = 'historic_memorial'][memorial = 'stele'][zoom >= 18],
+  [feature = 'historic_memorial'][memorial = 'stone'][zoom >= 18],
+  [feature = 'historic_memorial'][memorial = 'blue_plaque'][zoom >= 19],
+  [feature = 'historic_memorial'][memorial = 'plaque'][zoom >= 19],
+  [feature = 'historic_memorial'][zoom >= 19] {
```

Comment:
The last issue: I think this should be z17+, since we have no idea what can it be, so it's kind of generic.

Reply:
Are your sure for `memorial=blue_plaque` ? They seem to be not so large. Take a look at some examples from the [Guardian](https://www.theguardian.com/books/2009/nov/28/london-plaques-emily-cole-review).

I would put the same zoom level than stolperstein.

Reply:
I meant all the `memorial` values different than what we have on z18+ and z19+. Look at the similar case here:

https://github.com/gravitystorm/openstreetmap-carto/blob/9d16b996591c240080159ac9d11f69540d159637/amenity-points.mss#L474

Reply:
Is the fix satisfy you ?

Reply:
Thanks, it looks promising and I will test it soon. I just think that excluding statue is not necessary, because it will be rendered on z17+ anyway (that's why I told only about excluding z18+ and z19+ objects). It adds a bit more of clarity, so it's not an error, but I think this line is long enough.

Reply:
I have found the problem - this war memorial is not showing any icon at all (instead of a standard one), while the label is still visible from z17:

https://www.openstreetmap.org/node/3183626423#map=19/52.26992/20.99322

Reply:
I resolve both problems. The second one was related to a missing comma.